src/data/chbp_data_2019.json

```diff
@@ -204,24 +220,104 @@
           [
             [
               [
-                -122.318792211923395,
-                47.613843895634126
+                -122.31879221192,
+                47.61384389563
+              ],
+              [
+                -122.31879221192,
+                47.6139894133
+              ],
+              [
+                -122.31868574863,
+                47.6139894133
+              ],
+              [
+                -122.31868574863,
+                47.61384389563
+              ],
+              [
+                -122.31879221192,
+                47.61384389563
+              ],
+              [
+                -122.31879221192,
+                47.61384389563
+              ]
+            ]
+          ]
+        ]
+      }
+    },
+    {
+      "type": "Feature",
+      "properties": {
```

Comment:
Did you replace the exiting Point feature with this?

Reply:
Good catch, I did not remove the point for Chophouse Row. I've just added a commit removing that point:

<img width="288" alt="image" src="https://user-images.githubusercontent.com/33205894/61329583-83ef9c80-a7d2-11e9-87f5-3ed98d92ed6d.png">